app/src/main/java/dev/ukanth/iconmgr/favoriteIcons.java

```diff
@@ -0,0 +1,18 @@
+package dev.ukanth.iconmgr;
+
+import android.content.Intent;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import java.util.List;
+
+import dev.ukanth.iconmgr.dao.IPObj;
+
+public class favoriteIcons extends AppCompatActivity {
```

Comment:
Class names should not be camelCase. It should start with Capital "FavoriteIcons"